babel.config.js

```diff
@@ -12,5 +12,8 @@ module.exports = {
       },
     ],
   ],
-  plugins: ['@babel/plugin-proposal-class-properties'],
+  plugins: [
+    '@babel/plugin-proposal-class-properties',
+    '@babel/plugin-transform-flow-strip-types',
```

Comment:
I've been struggling with this issue https://github.com/react-native-community/upgrade-support/issues/152. There is probably a cleaner way to fix that, would love to get some guidance

Reply:
right, we could configure this through `env: {test: {...}}`

Reply:
Yep, good call, fixed in fa9a1c594fff47283a7a34cab38380f065b337ec